src/core/test/fuzzing/src/it/scala/DatabricksUtilities.scala

```diff
@@ -0,0 +1,260 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
+
+import java.io.FileInputStream
+import java.util.concurrent.TimeoutException
+
+import com.microsoft.ml.spark.FileUtilities.File
+import com.microsoft.ml.spark.SprayImplicits._
+import org.apache.commons.io.IOUtils
+import org.apache.http.client.methods.{HttpGet, HttpPost}
+import org.apache.http.entity.StringEntity
+import org.apache.http.impl.client.{CloseableHttpClient, HttpClientBuilder}
+import org.spark_project.guava.io.BaseEncoding
+import spray.json.DefaultJsonProtocol._
+import spray.json.{JsArray, JsObject, JsValue, _}
+
+import scala.concurrent.{ExecutionContext, Future, blocking}
+import scala.language.existentials
+import scala.sys.process.Process
+
+object DatabricksUtilities {
+  lazy val client: CloseableHttpClient = HttpClientBuilder.create().build()
+
+  // ADB Info
+  val region = "southcentralus"
+  val token = sys.env("MML_ADB_TOKEN")
+  val authValue: String = "Basic " + BaseEncoding.base64()
+    .encode(("token:" + token).getBytes("UTF-8"))
+  val baseURL = s"https://$region.azuredatabricks.net/api/2.0/"
+  val clusterName = "Test Cluster 2"
+  lazy val clusterId: String = getClusterIdByName(clusterName)
+  val folder = "/MMLSparkBuild/Build1"
+
+  //MMLSpark info
+  val topDir = new File(new File(getClass.getResource("/").toURI), "../../../../../../../")
```

Comment:
Eventually I plan to make some sbt magic to give the code a set of helpful env variables, but for now this is robust to where you run your code because its based on the target directory.